robot/basestation/static/js/elements/feed/camera-popup.js

```diff
@@ -0,0 +1,3 @@
+$(document).ready(() => {
+  $('.camera-feed').css('height', '720px');
+})
```

Comment:
@shayrin0 I kept it to 480 px as it was normally but hardcoded to 720px when it goes to the bigger page.